testsuite/MDAnalysisTests/analysis/test_bat.py

```diff
@@ -127,3 +128,11 @@ def test_bat_incorrect_dims(self, bat_npz):
         errmsg = 'Dimensions of array in loaded file'
         with pytest.raises(ValueError, match=errmsg):
             R = BAT(selected_residues, filename=bat_npz)
+
+    def test_bat_transformation(self, selected_residues, bat):
+        R = BAT(selected_residues)
+        pre_transformation = copy.deepcopy(bat[0])
+        R.Cartesian(bat[0])
+        assert_almost_equal(pre_transformation, bat[0], 5,
```

Comment:
```suggestion
        assert_almost_equal(pre_transformation, bat[0],
```

We don't want changes of lower precision to slide without notice -- we don't want any changes to the input at all.